src/js/routes/More/Donate.jsx

```diff
@@ -192,8 +194,8 @@ class Donate extends Component {
               <span className="u-no-break">Other Amount</span>
             </Button>
             {this.state.showCustomInput ? (
-              <span>
-                <InputGroup className="mb-3" style={{ width: '50%' }}>
+              <span style={{ display: 'flex', marginTop: 10, marginLeft: 20 }}>
```

Comment:
Hi @SailingSteve in general I think we should move away from inline styles like this, in favor of Styled components like this, where we replace the `<span` with `<DescriptionText`:
```
const DescriptionText = styled.p`
  font-size: 12px;
  margin: 0;
  @media print {
    font-size: 1.5rem;
  } 
`;
```